gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/LocalResourceManagerHelper.java

```diff
@@ -45,7 +53,25 @@
  * Utility to create a local Resource Manager mock for testing.
  *
  * <p>The mock runs in a separate thread, listening for HTTP requests on the local machine at an
- * ephemeral port.
+ * ephemeral port. While this mock attempts to simulate the Cloud Resource Manager, there are some
+ * divergences in behavior. The following is a non-exhaustive list of some of those behavioral
+ * differences:
+ *
+ * <ul>
+ * <li>This mock assumes you have adequate permissions for any action. Related to this,
+ *     <I>testIamPermissions</I> always indicates that the caller has all permissions listed in the
```

Comment:
Here and in the following: what about replacing `<I>` and `</I>` with `<i>` and `</i>` and adding missing `</li>`?